Backend/src/main/java/de/neuefische/backend/services/ProductService.java

```diff
@@ -27,15 +28,14 @@ public Product getProductById(String id) {
                 .orElseThrow(()->new RuntimeException("Product with id: " +id+ " not Found"));
```

Comment:
How are you going to handle this exception?

It might be easier to use a more specific exception type with only the id as argument instead of a complex message string.